rag_apps/agentic_rag_with_qwen_and_firecrawl/.env.example

```
@@ -0,0 +1,2 @@
+OPENAI_API_KEY=your_openai_api_key_here
+FIRECRAWL_API_KEY=your_firecrawl_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIRECRAWL_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)